content/en/docs/getting-started/running/index.md

```diff
@@ -267,7 +267,7 @@ docker run --rm -i -t \
 
 #### Least privileged
 
-Right now **is not officialy supported** but the following command should work great:
+The following capabilities should be enough to run Falco:
```

Comment:
What capabilities are we talking about? Below I can only see a command. Do you mean the following command?

Reply:
uhm good point i added them, ty!